cloudflared/rootfs/build.sh

```diff
@@ -40,3 +40,8 @@ chmod +x /usr/bin/cloudflared
 
 # Remove legacy cont-init.d services
 rm -rf /etc/cont-init.d
+
+# Remove s-6 legacy/deprecated (and not needed) services
```

Comment:
Can you elaborate on why we are doing this and if this will become irrelevant at some point?

Reply:
This is only necessary if we would run scripts from custom-cont-init.d or services.d which is in maintenance only mode since s6 version 3. We don't need it as we migrated to s6 v3 init scripts and services. I guess we can save som system resources not starting the daemons (but not sure tbh)